packages/empty-state/src/BasicEmptyState/BasicEmptyState.styles.ts

```diff
@@ -1,7 +1,6 @@
-import { css } from '@leafygreen-ui/emotion';
+import { css, cx } from '@leafygreen-ui/emotion';
 import { Theme } from '@leafygreen-ui/lib';
-import { palette } from '@leafygreen-ui/palette';
-import { spacing } from '@leafygreen-ui/tokens';
+import { color, spacing } from '@leafygreen-ui/tokens';
 
 export const rootStyles = css`
   padding: 40px;
```

Comment:
Nit: While we're updating to tokens, this could be updated as well if you want
```suggestion
  padding: ${spacing[1000]}px;
```

Reply:
^ yes please!